src/crimson/os/seastore/ordering_handle.h

```diff
@@ -0,0 +1,69 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
```

Comment:
```c++
// vim: ts=8 sw=2 smarttab expandtab
```

Reply:
I haven't seen anything about expandtab either.  Like I've remarked in the past, if you want to go through the existing crimson/ subdirectory and fix all of the existing files, go for it, but I see no advantage to introducing a new convention now.